main.tf

```diff
@@ -5,11 +5,45 @@ provider "aws" {
 resource "aws_s3_bucket" "demo-s3" {
   bucket = "${var.s3_bucket_prefix}-terraformcloud"
   versioning {
-    enabled = false
-    mfa_delete = false
```

Comment:
Accurics remediation:
Title: HIGH: S3 Buckets Server Side Encryption At Rest
Description:Vulnerability: Ensure that S3 Buckets have server side encryption at rest enabled to protect sensitive data./n/nRemediation:On the AWS S3 console, for each bucket that failed the rule, navigate to the Properties tab, and select Default encryption. Select an encryption method (AES-256 or AWS-KMS). Refer to: https://docs.aws.amazon.com/AmazonS3/latest/user-guide/default-bucket-encryption.html